lib/reline/line_editor.rb

```diff
@@ -138,7 +142,7 @@ def set_pasting_state(in_pasting)
 
   def reset(prompt = '', encoding:)
```

Comment:
The parameter `encoding:` can be also removed

Reply:
💦 Thanks! Removed!